astropy/coordinates/tests/test_arrays.py

```diff
@@ -41,10 +42,16 @@ def test_angle_arrays():
     npt.assert_almost_equal(a6.value, 945.0)
     assert a6.unit is u.degree
 
-    with pytest.raises(TypeError):
+    with ExitStack() as stack:
```

Comment:
I am just curious, I have seen `ExitStack` being added in this PR and another completely unrelated PR. Is this the new way to do `pytest.raises` or just context manager in general?

Reply:
It is no coincidence that you see it suddenly in two places - I saw that completely unrelated PR as well, and thought that it was a neat way to combine context managers, especially if, as here, the number of context managers included is not always the same (without it, I would have needed to use the "null context", which I don't really like - though better than using `ExitStack` when one has either one or zero context managers).